doc/integrator/filter.rst

```diff
@@ -43,6 +43,10 @@ If you wish to provide the filter functionality, proceed as follows:
   functionalities that all registered users shall have access to, via the
   ``functionalities.registered`` variable of your vars file.
 
+.. note::
+
+    Ngeo offer the filter functionnality only for singles layers and not for grouped layers.
```

Comment:
Ngeo offers the filter functionality only for single layers, not for grouped layers.